### PR TITLE
Reviewed nodes-zh.

### PR DESCRIPTION
**Pull request by Patrick-LuoYu**
Signed-off-by: Patrick-LuoYu <patrickluo@yunify.com>
/assign @Sherlock113

### CODE REVIEW

File: content/zh/docs/cluster-administration/nodes.md [hunk 1]
```diff
@@ -1,69 +1,72 @@
...
-keywords: "Kubernetes, KubeSphere, taints, nodes, labels, requests, limits"
+keywords: "Kubernetes, KubeSphere, 污点, 节点, 标签, 请求, 限制"
 description: "Kubernetes 节点管理"
 
 linkTitle: "节点管理"
 weight: 8100
 ---
 
-Kubernetes通过将容器放入容器组中并在节点上运行来运行工作负载。节点可以是虚拟机，也可以是物理机，这取决于集群环境。每个节点都包含运行容器组所需的服务，这些服务由控制平面管理。有关节点的更多信息，请参见[Kubernetes的官方文档](https://kubernetes.io/docs/concepts/architecture/nodes/)。
+Kubernetes 将容器放入容器组中并在节点上运行，从而运行工作负载。取决于具体的集群环境，节点可以是虚拟机，也可以是物理机。每个节点都包含运行容器组所需的服务，这些服务由控制平面管理。有关节点的更多信息，请参阅[ Kubernetes 官方文档](https://kubernetes.io/zh/docs/concepts/architecture/nodes/)。
```
Comment (Sherlock113) [inline content/zh/docs/cluster-administration/nodes.md:10]: Change all 容器组 to Pod in this guide.

**Comment by Patrick-LuoYu** [comment]
Fixed issues based on comments.
/assign @Sherlock113

File: content/zh/docs/cluster-administration/nodes.md [hunk 2]
```diff
@@ -1,69 +1,72 @@
...
+- **停止调度/启用调度**：您可以在节点重启或维护期间将节点标记为不可调度。Kubernetes 调度器不会将新 Pod 调度到标记为不可调度的节点。但这不会影响节点上现有工作负载。在 KubeSphere 中，您可以单击节点详情页面的**停止调度**将节点标记为不可调度。再次单击此按钮（**启用调度**）可将节点标记为可调度。
+- **标签**：您可以利用节点标签将 Pod 分配给特定节点。首先标记节点（例如，用 `node-role.kubernetes.io/gpu-node` 标记 GPU 节点），然后在[创建工作负载](../../project-user-guide/application-workloads/deployments/#步骤-5配置高级设置)时在**高级设置**中添加此标签，从而使 Pod 在 GPU 节点上运行。要添加节点标签，请单击**更多操作**，然后选择**编辑标签**。
    ![drop-down-list-node](/images/docs/cluster-administration/node-management-zh/drop-down-list-node.png)
 
    ![Label Node](/images/docs/cluster-administration/node-management-zh/label_node.png)
 
    ![Assign pods to nodes](/images/docs/cluster-administration/node-management-zh/assign_pods_to_node.png)
 
-- **污点**：污染允许节点排斥一系列的容器组。 您可以在节点详细信息页面上添加或删除节点污点。 要添加或删除污点，请单击**更多操作**，然后从下拉菜单中选择**污点管理**。
+- **污点**：污点允许节点排斥一些 Pod 。您可以在节点详情页面添加或删除节点污点。要添加或删除污点，请单击**更多操作**，然后从下拉菜单中选择**污点管理**。
```
Comment (Sherlock113) [inline content/zh/docs/cluster-administration/nodes.md:59]: 污点允许节点排斥一些 Pod 。
->
污点允许节点排斥一些 Pod。

**Review by Sherlock113 — COMMENTED** [review]
Have a look at my comments and make changes. Thanks.

**Comment by Sherlock113** [comment]
Change the location of all ZH-CN images to the zh-cn folder.

**Comment by Patrick-LuoYu** [comment]
Fixed issues based on comments.
/assign @Sherlock113

**Comment by Patrick-LuoYu** [comment]
Relocated images.
/assign @Sherlock113

**Pull request merged**